src/lwc/appointmentsCustomTable/appointmentsCustomTable.html

```diff
@@ -0,0 +1,75 @@
+<!--
+ - Created by Mariya on 02.02.2023.
+ -->
+
+<!-- Appointments Custom Table -->
+<template>
+    <div class="slds-theme-default slds-scrollable_y" style="height: 300px;">
+        <table aria-multiselectable="true"
+               class="slds-table slds-table_bordered slds-table_fixed-layout slds-table_resizable-cols" role="grid">
+            <thead>
+            <tr class="slds-line-height_reset">
+                <th class="slds-text-align_right slds-cell_action-mode" scope="col" style="width:3.25rem">
+                    <span id="column-group-header" class="slds-assistive-text">Choose a row</span>
+                    <div class="slds-th__action slds-th__action_form">
+                        <div class="slds-checkbox">
+                            <input type="checkbox" name="options" id="checkbox-unique-id-361"
+                                   value="checkbox-unique-id-361" tabindex="0"
+                                   aria-labelledby="check-select-all-label column-group-header"/>
+                            <label class="slds-checkbox__label" for="checkbox-unique-id-361"
+                                   id="check-select-all-label">
+                                <span class="slds-checkbox_faux"></span>
+                                <span class="slds-form-element__label slds-assistive-text">Select All</span>
+                            </label>
+                        </div>
+                    </div>
+                </th>
+                <th class="slds-is-resizable slds-is-sortable slds-cell_action-mode slds-cell-fixed" scope="col">
+                    Boutique
+                </th>
+                <th class="slds-is-resizable slds-is-sortable slds-cell_action-mode" scope="col">Customer</th>
+                <th class="slds-is-resizable slds-is-sortable slds-cell_action-mode" scope="col">Phone</th>
+                <th class="slds-is-resizable slds-is-sortable slds-cell_action-mode" scope="col">Email</th>
+                <th class="slds-is-resizable slds-is-sortable slds-cell_action-mode" scope="col">Purpose of Visit</th>
+                <th class="slds-is-resizable slds-is-sortable slds-cell_action-mode" scope="col">Date</th>
+                <th class="slds-is-resizable slds-is-sortable slds-cell_action-mode" scope="col">Start time</th>
+                <th class="slds-is-resizable slds-is-sortable slds-cell_action-mode" scope="col">Comments</th>
+            </tr>
+            </thead>
+            <tbody>
+            <template for:each={data} for:item="caseRecord">
+                <tr key={caseRecord.Id} class="slds-hint-parent">
```

Comment:
MINOR
Lets move <tr> to child component, smth like `tableRow`. So you iterate over components and communicate btw them.